frontend/express/public/javascripts/countly/countly.vue.components.js

```diff
@@ -372,4 +373,2886 @@
+                        </span>\
+                    </div>'
+    }));
+
+    Vue.component("cly-diff-helper", countlyBaseComponent.extend({
+        mixins: [
+            _mixins.i18n
+        ],
+        props: {
+            diff: {
+                type: Array
+            },
+        },
+        computed: {
+            hasDiff: function() {
+                return this.diff.length > 0;
+            },
+            madeChanges: function() {
+                return this.i18n("common.diff-helper.changes", this.diff.length);
+            }
+        },
+        methods: {
+            save: function() {
+                this.$emit("save");
+            },
+            discard: function() {
+                this.$emit("discard");
+            }
+        },
+        template: '<div class="cly-vue-diff-helper" v-if="hasDiff">\
+                        <div class="message">\
+                            <span class="text-dark">{{madeChanges}}</span>\
+                            <span class="text-light">{{ i18n("common.diff-helper.keep") }}</span>\
+                        </div>\
+                        <div class="buttons">\
+                            <cly-button :label="i18n(\'common.discard-changes\')" skin="light" class="discard-btn" @click="discard"></cly-button>\
+                            <cly-button :label="i18n(\'common.save-changes\')" skin="green" class="save-btn" @click="save"></cly-button>\
+                        </div>\
+                    </div>'
+    }));
+
+    Vue.component("cly-menubox", countlyBaseComponent.extend({
+        template: '<div class="cly-vue-menubox menubox-default-skin" v-click-outside="close">\
+                        <div class="menu-toggler" :class="{active: isOpened}" @click="toggle">\
+                            <div class="text-container">\
+                                <div class="text">{{label}}</div>\
+                            </div>\
+                            <div class="arrows-wrapper">\
+                                <div class="down ion-chevron-down"></div>\
+                                <div class="up ion-chevron-up"></div>\
+                            </div>\
+                        </div>\
+                        <div class="menu-body" v-show="isOpened">\
+                            <slot></slot>\
+                        </div>\
+                    </div>',
+        props: {
+            label: { type: String, default: '' },
+            isOpened: { type: Boolean, default: false }
+        },
+        methods: {
+            toggle: function() {
+                this.setStatus(!this.isOpened);
+            },
+            close: function() {
+                this.setStatus(false);
+            },
+            setStatus: function(targetState) {
+                this.$emit('status-changed', targetState);
+            }
+        }
+    }));
+
+    Vue.component("cly-button-menu", countlyBaseComponent.extend({
```

Comment:
Codacy found an issue: [Multiline support is limited to browsers supporting ES5 only.](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=6339769)